src/sentry/static/sentry/app/views/stream/sidebar.jsx

```diff
@@ -96,34 +97,39 @@ var StreamSidebar = React.createClass({
   render() {
     return (
       <div className="stream-sidebar">
-        <div className="stream-tag-filter">
-          <h6 className="nav-header">Text</h6>
-          <form onSubmit={this.onTextFilterSubmit}>
-            <input
-              className="form-control"
-              placeholder="Search title and culprit text body"
-              onChange={this.onTextChange}
-              value={this.state.textFilter}
-            />
-            {this.state.textFilter &&
-              <a className="search-clear-form" onClick={this.onClearSearch}>
-                <span className="icon-circle-cross" />
-              </a>
-            }
-          </form>
-          <hr/>
-        </div>
-
-        {_.map(this.props.tags, (tag) => {
-          return (
-            <StreamTagFilter
-              value={this.state.queryObj[tag.key]}
-              key={tag.key}
-              tag={tag}
-              onSelect={this.onSelectTag}
-            />
-          );
-        })}
+        {this.props.loading ?
+          <LoadingIndicator/> :
```

Comment:
Suggested alternative? I like it better than pulling the JSX into different statements inside `render`.